src/transactions/TransactionFrame.h

```diff
@@ -143,7 +144,8 @@ class TransactionFrame
     bool checkValid(Application& app, SequenceNumber current);
 
     // collect fee, consume sequence number
```

Comment:
mention transactions signed by priority whitelistees are not charged a fee

Reply:
It's explained at the point where we skip the fee adjustment.